_community/open-infra.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: default
 title: Apache    
```

Comment:
@jeffwilcox Should the title for both of these be Apache? I don't know about OpenInfra Foundation, but assume the page title should be that. Ignore if I am wrong.